test/Azure.Functions.Cli.Tests/E2E/InitTests.cs

```diff
@@ -508,5 +508,35 @@ public Task init_python_app_twice()
                 }
             }, _output);
         }
+
+        [Fact]
+        public Task init_python_app_generates_requirements_txt()
+        {
+            return CliTester.Run(new RunConfiguration
+            {
+                Commands = new[]
+                {
+                    "init . --worker-runtime python"
+                },
+                OutputContains = new[]
+                {
+                    "Writing requirements.txt"
+                },
+                CheckFiles = new FileResult[]
+                {
+                    new FileResult
+                    {
+                        Name = "requirements.txt",
+                        ContentContains = new []
+                        {
+                            "# DO NOT include azure-functions-worker in this file",
+                            "# The Python Worker is managed by Azure Functions platform",
+                            "# Manually managing azure-functions-worker may cause unexpected issues",
+                            "azure-functions"
```

Comment:
Could you please change the text to -

```python
# Do NOT include azure-functions-worker in this file
# The Python Worker is managed by Azure Functions platform
# Manually managing azure-functions-worker may cause unexpected issues
#
# If this file already contains azure-functions-worker dependency, kindly remove it
```

@anthonychu @ankitkumarr

Reply:
@anthonychu - Could you please help writing a friendlier text?

Reply:
Perhaps shorten it to something like “Do not include azure-functions-python-worker as it may conflict with the Azure Functions platform.”